.github/workflows/lint.yml

```diff
@@ -0,0 +1,24 @@
+name: Python application
+
+on:
+  pull_request
+
+jobs:
+  lint:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install flake8 black
+    - name: Lint
```

Comment:
Should this step run on *.py? ..I thought I saw that before but it disappeared

Reply:
It does its own search from .

```
mkdir -p foo/bar
echo "import a" > foo/foo.py
echo "import a" > foo/bar/bar.py
./foo/foo.py:1:1: F401 'a' imported but unused
./foo/bar/bar.py:1:1: F401 'a' imported but unused
```